app/helpers/patient_details_helper.rb

```diff
@@ -49,7 +49,7 @@ def linelist
       closed_at: closed_at&.rfc2822 || '',
       transferred_from: latest_transfer&.from_path || '',
       transferred_to: latest_transfer&.to_path || '',
-      expected_purge_date: updated_at.nil? ? '' : (updated_at + ADMIN_OPTIONS['purgeable_after'].minutes)&.rfc2822 || '',
+      expected_purge_date: monitoring || updated_at.nil? ? '' : (updated_at + ADMIN_OPTIONS['purgeable_after'].minutes)&.rfc2822 || '',
```

Comment:
What exactly is this meant to show in an export? true/false?

Reply:
Should be showing a date. I think there needs to be parentheses here:
```
(monitoring || updated_at.nil) ? ...
```

Reply:
Can't we just call the `expected_purge_date` function here? Maybe we should move that function definition inside of `patient_details_helper.rb` anyway. Also, it looks like this function is only really used as part of the `as_json` override call on the Patient model - doesn't appear to be used for exports at all. I think you need to update this line: https://github.com/SaraAlert/SaraAlert/blob/master/app/lib/import_export.rb#L646

Reply:
https://github.com/SaraAlert/SaraAlert/pull/581/commits/b790d1648ec0020abd631ac21fd212963815c239#diff-6a782ad659c7d687b8b004c762706d9c4c1ee24674305f8df71babe430abdb73R52